### PR TITLE
FLP-397 View Price Change Request

### PR DESCRIPTION
**Pull request by MikeV-TalentConsulting**

### CODE REVIEW

File: src/SFA.DAS.Apprenticeships.Domain/Apprenticeships/Api/PendingPriceChange.cs [hunk 1]
```diff
@@ -27,17 +28,12 @@ public static class PendingPriceChangeExtensions
 	{
 		public static InitiatedBy GetPriceChangeInitiatedBy(this PendingPriceChange pendingPriceChange)
 		{
-			if(pendingPriceChange.ProviderApprovedDate.HasValue && !pendingPriceChange.EmployerApprovedDate.HasValue) 
+			if(Enum.TryParse<InitiatedBy>(pendingPriceChange.Initiator, out var initiatedBy))
```
Comment (tom-gough) [inline src/SFA.DAS.Apprenticeships.Domain/Apprenticeships/Api/PendingPriceChange.cs:31]: can we change initiatedBy to initiator to match everywhere else?

File: src/SFA.DAS.Apprenticeships.Web.UnitTests/Controllers/ChangeOfPriceProviderControllerTests.cs [hunk 2]
```diff
@@ -129,37 +128,37 @@ public async Task ProviderInitiatedCheckDetailsPage_InvalidModel_ReturnsProvider
...
 
-			// Assert
-			var viewResult = result.ShouldBeOfType<ViewResult>();
-			viewResult.ViewName.Should().Be(ChangeOfPriceProviderController.ProviderEnterChangeDetailsViewName);
+            // Assert
+            var viewResult = result.ShouldBeOfType<ViewResult>();
+            viewResult.ViewName.Should().Be(ChangeOfPriceProviderController.ProviderEnterChangeDetailsViewName);
         }
-        
+
```
Comment (tom-gough) [inline src/SFA.DAS.Apprenticeships.Web.UnitTests/Controllers/ChangeOfPriceProviderControllerTests.cs:161]: I'm assuming all this tabbing looks right in the IDE?

File: src/SFA.DAS.Apprenticeships.Web/Extensions/UiExtensions.cs [hunk 3]
```diff
@@ -5,6 +5,11 @@ namespace SFA.DAS.Apprenticeships.Web.Extensions
 {
     public static class UiExtensions
     {
+        public static string FormatDate(this DateTime date)
```
Comment (tom-gough) [inline src/SFA.DAS.Apprenticeships.Web/Extensions/UiExtensions.cs:8]: I think there should already be GDS standard extensions for this (e.g. .ToGdsDate, .ToGdsFormatDate etc.)

**Review by tom-gough — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.Apprenticeships.Web/Extensions/UiExtensions.cs (hunk 3 above)
Comment (tom-gough) [reply src/SFA.DAS.Apprenticeships.Web/Extensions/UiExtensions.cs:8]: nvm this is actually in commitments only

File: src/SFA.DAS.Apprenticeships.Web.UnitTests/Controllers/ChangeOfPriceProviderControllerTests.cs (hunk 2 above)
Comment (MikeV-TalentConsulting) [reply src/SFA.DAS.Apprenticeships.Web.UnitTests/Controllers/ChangeOfPriceProviderControllerTests.cs:161]: Yes, it was all over the place

File: src/SFA.DAS.Apprenticeships.Domain/Apprenticeships/Api/PendingPriceChange.cs (hunk 1 above)
Comment (MikeV-TalentConsulting) [reply src/SFA.DAS.Apprenticeships.Domain/Apprenticeships/Api/PendingPriceChange.cs]: Done

**Review by tom-gough — APPROVED** [review]

**Pull request merged**